web/index.html

```diff
@@ -1463,19 +1474,24 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 	},
 
 	'disk.iotime': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of milliseconds spend by all operation. Read from fields 4 and 8.'
 	},
 	'disk.mops': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of merged operations. Read from fields 2 and 6.'
 	},
 	'disk.svctm': {
-		height: 0.5
+		height: 0.5,
+		info: 'Buisy time devided through the number of operations.'
 	},
 	'disk.avgsz': {
-		height: 0.5
+		height: 0.5,
+		info: 'Averange of data written or read per operation.'
 	},
 	'disk.await': {
-		height: 0.5
+		height: 0.5,
+		info: 'Averange of time needed for a operation.'
```

Comment:
The average time (in milliseconds) for I/O requests issued to the device to be served. This includes the time spent by the requests in queue and the time spent servicing them.

(taken from the iostat manpage)